### ISSUE TITLE
New Student redirect bug

### ISSUE THREAD

**Issue by karbassi** [CoderDojoChi/coderdojochi#569]
## The flow

1. New parent signs up for the website
2. They go to enroll in a class, but they don't have a student
3. They are redirected to "add new student" form.
4. Once they fill out the new student form, the `?next=XXXX` of this form is wrong and redirects them to the wrong URL.

This also happens when they have a student already but they want to add a new student, via the "add new student" button on the session detail page.

## Example
```
Page Not Found: http://localhost/classes/2/?enroll=True&student=4
```

What is happening here is that the parent clicked "add new student" button on the `/classes/2/` page. They were redirect to the form, filled it out, and submitted. The new student is not ID of 4, they should be the new created student ID, which could be 5 (if no one else has created a student before them).

## TLDR

The new student form isn't assigning the correct new student ID to the `?next`.

**Comment by karbassi** [comment]
@danmconrad did you have any time to look at this bug?
